app/queries/find_grandchildren.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class FindGrandchildren
+  def self.queries
+    [:find_grandchildren]
+  end
+
+  attr_reader :query_service
+  delegate :resource_factory, to: :query_service
+  delegate :orm_class, to: :resource_factory
+  def initialize(query_service:)
+    @query_service = query_service
+  end
+
+  def find_grandchildren(resource:)
+    query_service.find_members(resource: resource).flat_map do |child|
```

Comment:
With https://github.com/pulibrary/figgy/pull/397, the members could either be folders or boxes.  So we should probably filter the members to make sure we're only getting the members of boxes (and not getting FileSets too).  And then we'll want to get the directly-attached folders too.

Reply:
Oh, yeah, okay.

Reply:
sounds like `find_grandchildren` is no longer the correct method name.

Reply:
`find_children_and_grandchildren_of_type` doesn't have the same ring to it